examples/ethane.toml

```diff
@@ -0,0 +1,41 @@
+# run via:
+# cargo run ethane.toml --release
```

Comment:
Yepp, they both run the same simulation. I thought it would be good to show how to do the exact same thing using both methods.

We can change the `.rs` though if that makes more sense.